test/functional/specs/actions/redirectWithIdentity.spec.js

```diff
@@ -287,7 +287,7 @@ test("defaults to the first instance name", async () => {
   await extensionViewController.init({ extensionSettings });
   await instanceNamePicker.expectSelectedOptionLabel("alloy1");
   await extensionViewController.expectIsValid();
-  await extensionViewController.expectSettings({
+  await extensionViewController.expectSettingsToContain({
```

Comment:
The view is instantiated with a number of instances (alloy1, alloy2). When you get the settings, the object contains more properties than the instance name. I don't know how was this test passing before.

Reply:
The test seem to assert that the view opens with the `instanceName` field selected  by default with the first instanceName. 
I don't see the value of  this : `await extensionViewController.expectSettingsToContain({  instanceName: "alloy1"
  });` here.